createUserFeatureFlag/main.go

```diff
@@ -29,6 +29,8 @@ func processPutById(userId string, flagId string, featureFlagUserMapping models.
 
 	db := database.CreateDynamoDB()
 
+	utils.CheckRequestAllowed(db, utils.ConcurrencyDisablingLambda)
```

Comment:
1. Use a single dynamodb.AttributeValue map for the item. Instead of creating a new dynamodb.AttributeValue map for each attribute of the models.FeatureFlag object, you can create a single map and reuse it. This will reduce the overhead of creating new maps.
2. Use a dynamodb.PutItemInput pointer. Instead of creating a new dynamodb.PutItemInput instance, you can use a pointer to an existing instance. This will reduce the overhead of garbage collection.
3. Handle errors more gracefully. If there is an error creating the feature flag, you should log the error and return a more informative error message. For example, you could return a utils.FeatureFlagCreateError error with a message like "failed to create feature flag: error"

Reply:
@iamitprakash 
1. For the point 1 can I create a common method and invoke it by lambdas which are using it?